### ISSUE TITLE
TypeError: AlbaCLI.run() takes at least 2 arguments (2 given)

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework-alba-plugin#294]
We've observed that the `config` parameter is mandatory when performing a `AlbaCLI.run()` call. For some cases like `proxy-client-cfg` this is not mandatory.

```
In [2]: AlbaCLI.run(command="proxy-client-cfg", named_params={'host':'10.100.199.181', 'port':26204})
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-ce3de7dd4017> in <module>()
----> 1 AlbaCLI.run(command="proxy-client-cfg", named_params={'host':'10.100.199.181', 'port':26204})

TypeError: run() takes at least 2 arguments (2 given)
```

@khenderick created a small patch to get this working again:
```
In [4]: AlbaCLI.run(command="proxy-client-cfg", named_params={'host':'10.100.199.181', 'port':26204})
Out[4]: 
{u'cluster_id': u'mybackend-global-abm',
 u'node_cfgs': [[u'13covJSpGv7lptDY',
   {u'ips': [u'10.100.199.182'], u'port': 26412}],
  [u'Qeu0onUPO6odaG6o', {u'ips': [u'10.100.199.183'], u'port': 26412}],
  [u'jzA6kKHAKuHMcyLl', {u'ips': [u'10.100.199.181'], u'port': 26412}]],
 u'ssl_cfg': None,
 u'tcp_keepalive': {u'enable_tcp_keepalive': True,
  u'tcp_keepalive_intvl': 20,
  u'tcp_keepalive_probes': 3,
  u'tcp_keepalive_time': 20}}
```

**Comment by khenderick** [comment]
Fixed by @kinvaris by #295, packaged in openvstorage-backend-1.7.7-rev.846.79f6fa3

**Comment by kinvaris** [comment]
PASSED: Performed this on a node with given package:
```
In [4]: AlbaCLI.run(command="proxy-client-cfg", named_params={'host':'10.100.199.181', 'port':26204})
Out[4]: 
{u'cluster_id': u'mybackend-global-abm',
 u'node_cfgs': [[u'13covJSpGv7lptDY',
   {u'ips': [u'10.100.199.182'], u'port': 26412}],
  [u'Qeu0onUPO6odaG6o', {u'ips': [u'10.100.199.183'], u'port': 26412}],
  [u'jzA6kKHAKuHMcyLl', {u'ips': [u'10.100.199.181'], u'port': 26412}]],
 u'ssl_cfg': None,
 u'tcp_keepalive': {u'enable_tcp_keepalive': True,
  u'tcp_keepalive_intvl': 20,
  u'tcp_keepalive_probes': 3,
  u'tcp_keepalive_time': 20}}
```
